src/matchers/to-contain-element.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactTestInstance } from 'react-test-renderer';
+import { matcherHint, RECEIVED_COLOR as receivedColor } from 'jest-matcher-utils';
+import { checkHostElement, printElement } from './utils';
+
+export function toContainElement(
+  this: jest.MatcherContext,
+  container: ReactTestInstance,
+  element: ReactTestInstance | null
+) {
+  if (element !== null || !this.isNot) {
+    checkHostElement(element, toContainElement, this);
+  }
+
+  let matches = [];
```

Comment:
The check should mimic Jest DOM one: https://github.com/testing-library/jest-dom/blob/6d659995665076d78b9f874bec75545bfc56e5ab/src/to-contain-element.js#L6

Basically we want user to pass a host element taken from some other RNTL query, so we should do regular reference equality check: 
```
findAll((node) => noide === element)
```
That should work as, `ReactTestInstances` should have stable reference identity